### DISCUSSION TITLE
Ender3v2 not probing correctly -

### DISCUSSION THREAD

**Discussion by westmorland1066** [mriscoc/Ender3V2S1#891] [Q&A]
I've been running 3 E3v2 printers with mriscoc firmware for several months now without any real issues. I had been running the Ender3V2-422-BLTUBL-MPC-20221222.bin Last week, when running the tramming wizard, one of them started showing 0.0 for all the corners. It also went from doing two touches to only one at each corner. I tested the touch with a different cable on the problem machine (no change) and then moved the touch to on one of my other machines (worked perfectly), so I know it's not a problem with the touch itself or the cabling. I tried upgrading the firmware to the latest mriscoc version 20230312, no change, Then tried going back to the most recent Jyers release, again no change. Finally I went back to the latest official Ender release, which did give slightly different behavior: When I attempt to level it homes, then probes the front left corner, then goes back to center and stops.  I went back to the 20221222 version, still the same. It almost seems like a setting is stuck somewhere (I did try factory resetting, no effect) Has anyone run into this before? Any ideas how to fix it?

**Comment by TheWebMachine** [comment]
Have you tried reseating the main comm cable on the extruder assembly? I found that the release/retention clip on one side of mine comes loose way too easily despite looking locked in because the plastic catch below the visible surface is loose. Look at these pics...

Lifted slightly:
![20230425_135445](https://user-images.githubusercontent.com/1045052/234375584-8960ea62-c10b-4097-b503-61fd83c8ec83.jpg)

Correct:
![20230425_135455](https://user-images.githubusercontent.com/1045052/234375658-9795f3cb-8bcb-476d-94c6-d094d6870fc6.jpg)

**Reply by westmorland1066** [reply]
That looks like a different model printer.   The stock E3v2 doesn't have that.

**Reply by TheWebMachine** [reply]
Fair enough. I wasn't sure and had a similar issue before. 😌

**Comment by ArchieFoxer** [comment]
I had a similar issue and I fixed it by setting a smaller mesh inset
X min = 35
X max = 195
Y min = 35
Y max = 195
